### DISCUSSION TITLE
Is it possible to create a tunnel for choosing the products ?

### DISCUSSION THREAD

**Discussion by bennukem** [saleor/saleor#12311] [Q&A]
Hi guys,
Is it possible to create a tunnel for choosing the products ?
Scenario : 
 choose a box of 4 or 6 then have to choose the 4/6 products for that box.

**Comment by timuric** [comment]
@bennukem not sure what do you mean by tunnel, but from the description it sounds like this is front end logic

**Reply by bennukem** [reply]
It s about to fallow a scenario.
Have to fallow steps depend on the previous one.
My idea is for example a set in restaurant.
You select Menu A -> choose your mean dish -> choose your sidedish -> depend on the sidedish , which sauce -> choose a drink -> ... -> and finally can checkout

Mc donalds kiosk clone is something I am looking for.
